main.go

```diff
@@ -133,6 +140,9 @@ func main() {
 </body>
 </html>
 						`))
+		if err != nil {
```

Comment:
this doesn't really add anything useful, I think it's ok to ignore err here (and remove it from line 127).